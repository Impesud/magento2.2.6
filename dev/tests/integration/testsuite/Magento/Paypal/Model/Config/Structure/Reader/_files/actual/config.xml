version https://git-lfs.github.com/spec/v1
oid sha256:3f66bb27e1c8e6d2ed54d20cb1b61d4cff7839aa1a57271b85e932658cd167dc
size 27147
