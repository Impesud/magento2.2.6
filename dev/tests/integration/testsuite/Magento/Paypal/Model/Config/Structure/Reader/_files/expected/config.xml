version https://git-lfs.github.com/spec/v1
oid sha256:1e4afc19f1a198a5c7bf7ff8e8b3df12d0a42825f7427d726bef95e6179b1cec
size 212147
